localstripe/localstripe-v3.js

```diff
@@ -80,20 +80,49 @@ function openModal(text, confirmText, cancelText) {
 }
 
 class Element {
-  constructor() {
+  constructor(elements) {
     this.listeners = {};
+    // Element needs a reference to the object that created it, in order to
+    // thoroughly destroy() itself.
+    this._elements = elements;
   }
 
+  // _state tracks calls to unmount, mount, and destroy, and is always one
+  // of 'unmounted', 'mounted', or 'destroyed'.
+  _state = 'unmounted';
+
   mount(domElement) {
     if (typeof domElement === 'string') {
       domElement = document.querySelector(domElement);
+    } else if (!(domElement instanceof window.Element)) {
+      throw new Error('Invalid DOM element. Make sure to call mount() with ' +
+          'a valid DOM element or selector.');
```

Comment:
It's just visual style, but the rest of the file uses this kind of indentation:
```suggestion
      throw new Error('Invalid DOM element. Make sure to call mount() with ' +
                      'a valid DOM element or selector.');
```